lms/templates/edxnotes/edxnotes.html

```diff
@@ -111,11 +110,11 @@ <h2 class="placeholder-title">${_('You have not made any notes in this course ye
     <%block name="js_extra">
     <%static:require_module module_name="js/edxnotes/views/page_factory" class_name="NotesPageFactory">
         NotesPageFactory({
-            disabledTabs: ${disabled_tabs},
-            notes: ${notes},
-            notesEndpoint: '${notes_endpoint}',
-            pageSize: '${page_size}',
-            debugMode: ${debug}
+            disabledTabs: ${disabled_tabs | n, dump_js_escaped_json},
+            notes: ${dump_js_escaped_json(notes, NoteJSONEncoder) | n},
+            notesEndpoint: ${notes_endpoint | n, dump_js_escaped_json},
+            pageSize: '${page_size | n, dump_js_escaped_json}',
```

Comment:
Typically, if you use this you wouldn't have the quotes around it.  If you want it to be a string, then you would use `js_escaped_string`.  For this example, seems like maybe just dropping the quotes?